docs/contributor/devenv_setup.md

```diff
@@ -273,3 +276,32 @@ Repo Name: Red Hat Enterprise Linux 9 for x86_64 - AppStream (RPMs)
 Repo URL:  https://cdn.redhat.com/content/dist/rhel9/$releasever/x86_64/appstream/os
 Enabled:   1
 ```
+
+### Increase Virtual Machine Disk Size
+
+Log into the hypervisor host, **shut down the virtual machine** and run the
+following commands to resize its disk.
+```
+VM_NAME=microshift-dev
+VM_DISK=/var/lib/libvirt/images/${VM_NAME}.qcow2
+INCREASE_BY=20
+
+sudo qemu-img resize ${VM_DISK} +${INCREASE_BY}G
+sudo virsh start ${VM_NAME}
```

Comment:
If you include `start`, why not stop (...`destroy`...)? It's mentioned in the desc above the snippet, but I'd include that personally